tests/dotnet/UnitTests/ProjectTest.cs

```diff
@@ -431,6 +431,38 @@ public void IsNotMacBuild (ApplePlatform platform, string runtimeIdentifiers, bo
 			}
 		}
 
+		[Test]
+		[TestCase (ApplePlatform.MacCatalyst, "maccatalyst-arm64;maccatalyst-x64")]
+		public void IsOverrideRuntimeIdentifier (ApplePlatform platform, string runtimeIdentifiers)
+		{
+			var project = "MySimpleApp";
+			Configuration.IgnoreIfIgnoredPlatform (platform);
+			Configuration.AssertRuntimeIdentifiersAvailable (platform, runtimeIdentifiers);
+			var project_path = GetProjectPath (project, runtimeIdentifiers: runtimeIdentifiers, platform: platform, out var appPath);
+			Clean (project_path);
+			var properties = GetDefaultProperties (runtimeIdentifiers);
+			properties ["RuntimeIdentifier"] = "maccatalyst-arm64";
```

Comment:
Minor: building for x64 is faster than for arm64, so this speeds up the test a little bit:

```suggestion
			properties ["RuntimeIdentifier"] = "maccatalyst-x64";
```